lib/cprofiles/src/cprof_profile.c

```diff
@@ -307,4 +307,4 @@ int cprof_profile_add_comment(struct cprof_profile *profile, int64_t comment)
     profile->comments_count++;
 
     return 0;
-}
```

Comment:
please do not touch this file

Reply:
Good catch. I had to manually apply local changes to cprof files to make the build and run the unit test. I then manually reverted those changes but still got the one line diff, which I don't still understand.

Do maintainers have the permission to revert this file change?

Reply:
It turns out there is an additional newline character appended by my vim editor. `:set noeol` fixes the one diff. See https://stackoverflow.com/questions/1050640/how-to-stop-vim-from-adding-a-newline-at-end-of-file